app.py

```diff
@@ -2,45 +2,50 @@
 import time
 
 
-class Game:
-
-    @staticmethod
-    def choose_language(choice):
-        choice = choice.upper()
-        if choice == "ENGLISH":
-            return f"{choice} Language Selected."
+class TestTypingSpeed:

```

Comment:
I think this class should have a constructor as well, to give the client the option to select a language when instantiating the class. If no option is provided, the choose_language method should be called to set the choice attribute of the class.